develop/core/tests/test_invoice.py

```diff
@@ -239,6 +238,27 @@ def test_order_item_no_trial_amount(self):
         self.assertEqual(self.new_invoice.get_discounts(), 0)
         self.assertEqual(self.new_invoice.order_items.first().trial_amount, month_offer.current_price())
 
+    def test_invoice_global_discount(self):
+        month_offer = Offer.objects.get(pk=6)
+        self.new_invoice.global_discount = 10
+        self.new_invoice.save()
+        self.new_invoice.add_offer(month_offer)
+        self.assertEqual(self.new_invoice.total, month_offer.current_price() - self.new_invoice.global_discount)
+
+    def test_invoice_negative_global_discount(self):
+        month_offer = Offer.objects.get(pk=6)
+        self.new_invoice.global_discount = -10
+        self.new_invoice.save()
```

Comment:
cool, but fmi - what's the use case for a **negative** discount?

Reply:
To check that math.fabs() is doing it's job. Don't want to get caught in a scenario where you might endup adding or substracting more depending on what the user inputed. It should work in either positive or negative values